guides/_subsections/navigate-the-dashboard.md

```diff
@@ -14,67 +14,67 @@ Dashboard cards are widgets that display information about your Cryostat instanc
 <ol>
   <li>
     {% capture target-jvm-details-text %}
-    The <i>Target JVM Details</i> card displays information about the target JVM application that is currently selected. There are two tabs that display different information:
+    The <i>Target JVM Details</i> <code>card</code> displays information about the <code>target</code> <b>JVM</b> application that is currently selected. There are two tabs that display different information:
     <ul>
         <li>
-            <b>Details tab</b> - Displays information about the target JVM application, including:
+            <i>Details Tab</i> - Displays information about the <code>target</code> <b>JVM</b> application, including:
```

Comment:
target applications are referenced in multiple other files, but they are never in code blocks. It would be best to make it consistent whether by putting it in code blocks elsewhere or taking it out here.